src/main/java/wf/bitcoin/javabitcoindrpcclient/BitcoinJSONRPCClient.java

```diff
@@ -1704,11 +1714,6 @@ public Out getTransactionOutput() {
       public String scriptPubKey() {
         return mapStr("scriptPubKey");
       }
-
-      @Override
-      public String address() {
-          return mapStr("address");
-      }
```

Comment:
obsolete method?

Reply:
Yes, this only works for bitcoind nodes of which version is under 0.15 (see above). Also, now that we provide general map wrapper access it is no longer necessary to to provide a typed API for stuff that is non-standard.